stable/traefik/values.yaml

```diff
@@ -101,6 +101,7 @@ ssl:
   defaultIPList: []
     # - 1.2.3.4
   # cipherSuites: []
+  customSecretName: ""
```

Comment:
This should be optional (commented out by default)

Reply:
I'll update as suggested

Reply:
I'll update as suggested